pandas/core/generic.py

```diff
@@ -4340,7 +4340,7 @@ def sample(self, n=None, frac=None, replace=False, weights=None,
             being sampled.
             If weights do not sum to 1, they will be normalized to sum to 1.
             Missing values in the weights column will be treated as zero.
-            inf and -inf values not allowed.
+            `inf` and `-inf` values not allowed.
```

Comment:
If this is code literals, then it should be double-backticks and quoted

    ``'inf'``

which is a bit noise. How about "infinite values are not allowed.?